README.md

```diff
@@ -73,7 +73,7 @@ To contribute to the development of xDSL follow the subsequent steps.
 git clone https://github.com/xdslproject/xdsl.git
 cd xdsl
 # set up the venv and install everything
-make venv
+make
```

Comment:
`make venv` did not work for me on initial clone for some reason

Reply:
CC @AntonLydike

Reply:
Yes, I saw this yesterday, #2316 had a rename from `venv` to `venv/`, which broke the target silently, for now one needs to do `make venv/` to create the venv which is suboptimal.

Reply:
Opened #2419 to fix this!

Reply:
@knickish  Once Anton's change is merged, could you please revert this part of the change, so we can merge the URL fixes?

Reply:
sure, will do that

Reply:
ok dropped that commit